[blog] Add blog on Pulsar Summit CFP (#5888)
apache/pulsar: site2/website/blog/2019-12-18-Pulsar-summit-cfp.md

---
author: Jennifer Huang  
authorURL: https://twitter.com/Jennife06125739  

> title: Pulsar Summit SF 2020 CFP is now open

Reviewer: ```suggestion
title: Pulsar Summit San Francisco 2020 CFP is now open
```

---

Pulsar Summit is an annual conference dedicated to Apache Pulsar community, bringing together an international audience of CTOs/CIOs, developers, data architects, data scientists, Apache Pulsar committers/contributors, and the messaging and streaming community, to share experiences, exchange ideas and knowledge about Pulsar and its growing community, and receive hands-on training sessions led by Pulsar experts.

We are excited to announce that the first Pulsar Summit will be held in San Francisco on April 20th, 2020. Talk submissions, pre-registration, and sponsorship opportunities are now open for the conference!

## Speak at Pulsar Summit
Presentations and lightning talks are accepted for speaking proposals. Suggested topics cover Pulsar use cases, operations, technology deep dive, and ecosystem. Submissions are open until **January 31, 2020**.

> If you are unsure about your proposal, or want some feedback or advice in general, please do not hesitate to reach out to [sf-2020@pulsar-summit.org](mailto:sf-2020@pulsar-summit.org). We are happy to help out! Further details are available on the [CFP website](https://pulsar-summit.netlify.com/call-for-presentations/).

Reviewer: ```suggestion
If you are unsure about your proposal, or want some feedback or advice in general, please do not hesitate to reach out to [sf-2020@pulsar-summit.org](mailto:sf-2020@pulsar-summit.org). We are happy to help out! Further details are available on the [CFP website](https://pulsar-summit.org/call-for-presentations/).
```

## Dates to remember
- CFP opens: December 15, 2019
- CFP closes: January 31, 2020 - 23:59 PST
- CFP notification: February 21, 2020
- Schedule announcement: February 24, 2020

## Speaker benefits
When your speaking proposal is approved, you will enjoy the following benefits:

- Full conference pass.
- Exclusive swag only available to speakers.
- Expand your network and raise your profile in the Pulsar community.
- A professionally produced video of your presentation.
- Your name, title, company, and bio will be featured on the Pulsar Summit San Francisco 2020 website.
- Your session will be added to the Pulsar Summit YouTube Channel and promoted on Twitter and LinkedIn.

## Pre-registration

> If you are interested in attending Pulsar Summit San Francisco 2020, we’d like to hear from you with your [pre-registration](https://pulsar-summit.netlify.com/pre-registration/). Your ideas are very important to us, and we will prepare the content accordingly. 

Reviewer: ```suggestion
If you are interested in attending Pulsar Summit San Francisco 2020, we’d like to hear from you with your [pre-registration](https://pulsar-summit.org/pre-registration/). Your ideas are very important to us, and we will prepare the content accordingly. 
```
